spmd/tensor/ops/pointwise_ops.py

```diff
@@ -28,10 +28,20 @@
 pointwise_ops = [
     # please keep the entries below alphabetically sorted
     "aten.abs.default",
+    "aten.acos.Scalar",
```

Comment:
I think these op overloads are not actually going through `__torch_dispatch__`, so even if we added there they might not really being used, they are getting preprocessed and passed to a op that goes through dispatch op (i.e. aten.cos.default). For ops that are actually going through, they are mainly the ops in native_functions.yaml and have a `dispatch` bounded method https://github.com/pytorch/pytorch/blob/e217b30b0fc98df226654cef0617eb41a177531f/aten/src/ATen/native/native_functions.yaml#L474
`